Week4/ReadingList/app/src/main/java/com/florencenjeri/readinglist/BooksViewHolder.kt

```diff
@@ -0,0 +1,32 @@
+package com.florencenjeri.readinglist
+
+import android.view.View
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.florencenjeri.readinglist.model.Books
+import kotlinx.android.synthetic.main.book_list_item.view.*
+
+class BooksViewHolder(view: View) : RecyclerView.ViewHolder(view) {
+
+    private var bookTitle = view.bookTitle
```

Comment:
Once again you don't really need these properties, you can just use them directly in the bindPhoto function.

I'd also rename the `bindPhoto` to `setData` as it sets different types of data to the UI not just the photo!